### PR TITLE
Feature/task 1 core concepts

### PR DESCRIPTION
**Pull request by vposidailo**
Splitted Movies to backend and frontend folders.

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "Movie backend",
+  "main": "index.js",
+  "scripts": {
```
Comment (EvgeniyTT) [inline backend/package.json:6]: Please add "start" script. It's a crucial part of any js app. Anyone who runs it shouldn't guess how to do this. Just "npm start" and everything should launch. Even you after 3 weeks will forget how to start it without a good "script" section :)

File: frontend/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "author": "Vitalii Posidailo",
+  "license": "ISC",
+  "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.5",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "express": "^4.16.4",
```
Comment (EvgeniyTT) [inline frontend/package.json]: you can probably remove "express" from this file

File: frontend/src/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Search from './components/search';
+import MovieTileItem from './components/movie-tile-item';
+
+// create element examples
+const header = React.createElement('div', null, React.createElement('h1', null, 'This is a header'));
+const middle = React.createElement('div', null, 'here should be search bar');
+const footer = React.createElement('div', null, 'Footer');
+
```
Comment (EvgeniyTT) [inline frontend/src/index.js:11]: I see "Warning: Each child in a list should have a unique "key" prop." in a console...

File: frontend/src/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+// create function component examples
+const movies = movie => <div><h4>{ movie.title }</h4><p>{movie.author}</p><p>{movie.duration}</p></div>;
```
Comment (EvgeniyTT) [inline frontend/src/index.js]: If it's "movies" then the function should, probably, accept an array as an argument

File: backend/index.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+const express = require('express')
+const app = express()
+app.get('/', (req, res) => res.send('Movie backend'))
+app.listen(3000, () => console.log('Listening on port 3000!'))
```
Comment (EvgeniyTT) [inline backend/index.js]: both front and back are run on the port 3000, let's move BE to another port?

**Review by EvgeniyTT — CHANGES_REQUESTED** [review]

File: frontend/src/index.js (hunk 4 above)
Comment (vposidailo) [reply frontend/src/index.js]: Not clear what do you mean?

File: backend/package.json (hunk 1 above)
Comment (vposidailo) [reply backend/package.json:6]: Done.

File: backend/index.js (hunk 5 above)
Comment (vposidailo) [reply backend/index.js]: Done.

File: frontend/package.json (hunk 2 above)
Comment (vposidailo) [reply frontend/package.json]: Done.

File: frontend/src/index.js (hunk 3 above)
Comment (vposidailo) [reply frontend/src/index.js:11]: Done.

File: frontend/src/index.js (hunk 4 above)
Comment (EvgeniyTT) [reply frontend/src/index.js]: I thought about  something like:

```
const movies = movies => movies.map( movie => (
<div>
  <h4>{ movie.title }</h4>
  <p>{movie.author}</p>
  <p>{movie.duration}</p>
</div>));
```

but that's a small thing, anyway, this code will be changed soon.

**Pull request merged**